modin/numpy/utils.py

```diff
@@ -19,12 +19,7 @@
 _INTEROPERABLE_TYPES = (pd.DataFrame, pd.Series)
 
 
-def try_convert_from_interoperable_type(obj):
+def try_convert_from_interoperable_type(obj, copy=True):
     if isinstance(obj, _INTEROPERABLE_TYPES):
-        new_qc = obj._query_compiler.reset_index(drop=True)
-        new_qc.columns = range(len(new_qc.columns))
-        obj = np.array(
-            _query_compiler=new_qc,
-            _ndim=2 if isinstance(obj, pd.DataFrame) else 1,
-        )
```

Comment:
why is this no longer needed?

Reply:
Since we now natively accept modin objects in the modin array constructor, we can just call the constructor. We don't need the reset index and columns part since thats handled in the constructor as well.